src/styles/_catalog.scss

```
@@ -20,3 +19,21 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width: 488px) {
```

this all @media styling should be placed in `.catalog` class line 6, please move them up and then you can remove .catalog class from inside, it will automatically points to this class :) 

```suggestion
@media (min-width: 488px) {
     grid-template-columns: repeat(2, 200px);
 }
```